src/components/SEO.js

```diff
@@ -49,6 +50,14 @@ function SEO({ description, lang, meta, title, type }) {
 					name: 'twitter:card',
 					content: 'summary'
```

Comment:
Check out Twitter documentation here:

* https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary
* https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image

We have twitter:card set to `summary` right now, which requires a 1:1 aspect ratio image. If we want to use the banner (closer to 2:1), then we should instead set it to be `summary_large_image`.

You can use the [Twitter validator](https://cards-dev.twitter.com/validator) to check if it worked. Right now, it looks like

![image](https://user-images.githubusercontent.com/1538624/107906482-3c397580-6f1f-11eb-9643-b588bd111148.png)

Reply:
we can't get it to work on twitter rip